SpatialGDK/Source/Private/Interop/SpatialSender.cpp

```diff
@@ -221,6 +224,37 @@ void USpatialSender::SendComponentUpdates(UObject* Object, USpatialActorChannel*
 	}
 }
 
+
+void USpatialSender::FillComponentInterests(FClassInfo* Info, bool bNetOwned, TArray<Worker_InterestOverride>& ComponentInterest)
+{
+	Worker_InterestOverride SingleClientInterest = { Info->SingleClientComponent, bNetOwned };
+	ComponentInterest.Add(SingleClientInterest);
+
+	Worker_InterestOverride HandoverInterest = { Info->HandoverComponent, false };
+	ComponentInterest.Add(HandoverInterest);
+}
+
+void USpatialSender::SendComponentInterests(AActor* Actor, Worker_EntityId EntityId)
+{
+	check(!NetDriver->IsServer());
+
+	// This effectively checks whether the actor is owned by our PlayerController
+	bool bNetOwned = Actor->GetNetConnection() != nullptr;
+
+	TArray<Worker_InterestOverride> ComponentInterest;
+
+	FClassInfo* ActorInfo = TypebindingManager->FindClassInfoByClass(Actor->GetClass());
+	FillComponentInterests(ActorInfo, bNetOwned, ComponentInterest);
+
+	for (UClass* SubobjectClass : ActorInfo->SubobjectClasses)
```

Comment:
Also couldn't this be encapsulated into `FillComponentInterests`? As in it iterates through all the Subobjects as well since it doesn't look like we ever do one of the other.